paddle/fluid/inference/tests/api/tester_helper.h

```diff
@@ -41,7 +41,12 @@ DEFINE_string(model_name, "", "model name");
 DEFINE_string(infer_model, "", "model path");
 DEFINE_string(infer_data, "", "data file");
 DEFINE_string(refer_result, "", "reference result for comparison");
-DEFINE_int32(batch_size, 1, "batch size.");
+DEFINE_int32(batch_size, 1, "batch size");
+DEFINE_int32(warmup_batch_size, 100, "batch size for quantization warmup");
+DEFINE_int32(iterations, 0, "number of batches to process");  // setting to 0,
+                                                              // means process
+                                                              // the whole
+                                                              // dataset
```

Comment:
comment from 46-49 should be in one line.

Reply:
Done.